doc/glossary.rst

```diff
@@ -267,18 +267,17 @@
                 meaning is intended. 
 
 	OSD fsid
-                This is a unique identifier used to further improve the
-                uniqueness of an OSD and it is found in the OSD path in a file
-                called ``osd_fsid``. This ``fsid`` term is used interchangeably
-                with ``uuid``
+                This is a unique identifier used to identify an OSD. It is
+                found in the OSD path in a file called ``osd_fsid``. The
+                term ``fsid`` is used interchangeably with ``uuid``
```

Comment:
The new phrasing seems potentially confusing to me.  I was not previously aware of ``osd_fsid``, probably because (unlike the journal UUID) I'd never had it go wrong or had to use it.  So in the last sentence, ``fsid`` to me is synonymous with what is elsewhere called ``cluster_fsid``, e.g. as reported by ``ceph -s`` -- and which doesn't seem to be in this glossary at all. And since ``UUID`` is a structured unique identifier format that is not specific to Ceph, the last sentence is more confusing.

Whatever ``osd_fsid`` is, I wish it'd be named something different, but here we are.

Reply:
I don't disagree with any of this, but I can't address it in this backport so I have to ask you to approve this PR so that I can open another PR that begins the process of addressing these problems.

The terms "OSD fsid", "OSD id", and "OSD uuid" are used throughout this documentation in a way that, if not absolutely interchangeable, suggests that three different parties at three different times had in mind the same thing when they used one of these three terms to refer to the thing that they had in mind. Cleaning this up is a project that will span probably three to five more pull requests.

I have written an email to Ernesto Puerta, asking him to provide me with word-of-God definitions of each of these terms. But it's the Christmas break, and I don't expect a response before the fifth of January. 

I want to give you first right of refusal over any changes that are made in this connection, because you have a finer sense of this part of Ceph than anyone else working on the docs. So, with your permission, I will instruct anyone who works on this to wait until you have been consulted and satisfied before any future changes are merged into "main". 

So: give me an approval on this PR so that I can get to work on these changes in a non-backport PR.

Reply:
@anthonyeleven https://github.com/ceph/ceph/pull/49593